cmd/livepeer_cli/wizard_transcoder.go

```diff
@@ -185,3 +189,69 @@ func (w *wizard) callReward() {
 	fmt.Printf("Calling reward for round %v\n", c)
 	httpGet(fmt.Sprintf("http://%v:%v/reward", w.host, w.httpPort))
 }
+
+func (w *wizard) vote() {
+	if w.offchain {
+		glog.Error("Can not vote in 'offchain' mode")
+		return
+	}
+
+	fmt.Print("Enter the contract address for the poll you want to vote in")
+	poll := w.readStringAndValidate(func(in string) (string, error) {
+		if !ethcommon.IsHexAddress(in) {
+			return "", fmt.Errorf("invalid hex address address=%v", in)
+		}
+		return in, nil
+	})
+
+	var (
+		confirm = "n"
+		choice  = types.VoteChoice(-1)
+	)
+
+	for confirm == "n" {
+		choice = types.VoteChoice(-1)
+		choices := w.showVoteChoices()
+		if choices == nil {
+			return
+		}
+
+		for ok := choice.IsValid(); !ok; {
+			fmt.Printf("Enter the ID of the option you want to vote for")
+			choice = types.VoteChoice(w.readInt())
+			if ok := choice.IsValid(); ok {
+				break
+			}
+			fmt.Println("Must enter a valid ID")
+		}
+
+		fmt.Printf("Are you sure you want to vote \"%v\" ? (y/n)", choice.String())
+		confirm = w.readStringYesOrNo()
+	}
+
+	data := url.Values{
+		"poll":     {poll},
+		"choiceID": {fmt.Sprintf("%v", int(choice))},
+	}
+
+	fmt.Println(data)
```

Comment:
Seems like this is an extraneous print statement?

Reply:
oh yeah sorry I forgot to remove this

Reply:
fixed in 2857efe